### PR TITLE
Talk list API

### PR DESCRIPTION
**Pull request by setmao**
## Types of changes
**Thanks for sending a pull request! Please fill in the following content to let us know better about this change.**
Please put an `x` in the box that applies

- [ ] **Bugfix**
- [X] **New feature**
- [ ] **Refactoring**
- [ ] **Breaking change** (any change that would cause existing functionality to not work as expected)
- [ ] **Documentation Update**
- [ ] **Other (please describe)**

## Description
**Describe what the change is**
You don't need to explain HOW you change the code. Your code, including comments in the code should explicitly
self-describe enough about how you change the code.

## Related Issue
If applicable, refernce to the issue related to this pull request.

## More Information
**Screenshots**

**Additional context**
Add any other context about the problem here. You may also want to refer
to [how to wirte the perfect pull request](https://github.blog/2015-01-21-how-to-write-the-perfect-pull-request/)

### CODE REVIEW

File: src/events/api/views.py [hunk 1]
```diff
@@ -1,10 +1,25 @@
...
 
 
 class TalkDetailAPIView(RetrieveAPIView):
 
-    queryset = models.TalkProposal.objects.all()
+    queryset = TalkProposal.objects.all()
     serializer_class = serializers.TalkDetailSerializer
+
+
+class TalkListAPIView(views.APIView):
```
Comment (Maliaotw) [inline src/events/api/views.py:16]: import `ListAPIView` and replace `views.APIView`.

File: src/events/api/views.py [hunk 2]
```diff
@@ -1,10 +1,25 @@
...
+
+    queryset = TalkProposal.objects.filter_accepted()
+    serializer_class = serializers.TalkListSerializer
+
+
+class SponsoredEventListAPIView(views.APIView):
```
Comment (Maliaotw) [inline src/events/api/views.py:22]: import `ListAPIView` and replace `views.APIView`.

**Comment by mattwang44** [comment]
@setmao please remember to fix the flake8 warning!

**Review by Maliaotw — APPROVED** [review]
LGTM

**Pull request merged**